dist/css/bootstrap.css

```diff
@@ -4415,8 +4415,7 @@ select[multiple].input-lg {
   }
   .form-inline .control-label {
     margin-bottom: 0;
-    vertical-align: -webkit-calc(-2px);
-    vertical-align:         calc(-2px);
+    vertical-align: calc(-2px);
```

Comment:
There are a few of these that just seem entirely obsolete

Reply:
lol true